packages/radix/src/components/Heading.tsx

```diff
@@ -28,6 +28,22 @@ export const Heading = React.forwardRef<HTMLHeadingElement, HeadingProps>((props
         },
       },
       variants: {
+        fontWeight: {
+          400: {
+            text: {
+              normal: {
+                fontWeight: 400,
+              },
+            },
+          },
+          500: {
+            text: {
+              normal: {
+                fontWeight: 500,
+              },
+            },
+          },
+        },
```

Comment:
#226 sets default Heading weight to `500`. I pictured that after merging my PR the weight changes should happen via `sx` prop.

Reply:
Interesting, doesn't it make more sense via a variant though? Especially because with sx it's free for all, you can't enforce the values.

Reply:
Yeah, we could do a variant, but we've been using medium font weight quite consistently lately.

The problem with weight variant is that Regular needs looser letter spacing depending on the size used. This needs size × variant combos, which are removed in #226 because they aren't possible in Modulz app yet, which will be a problem when Radix is generated by Modulz.

Also, purely from the design perspective, I think consistent headings weight is good.

Reply:
So medium weight is the go-to now, and exceptions are treated on case-by-case basis.

Reply:
Ok @vladmoroz , so just so I understand what I need to change here then, are we saying remove the fontWeight variant alltogether and update all the cases where it was used to remove it because once your PR goes in, it'll be medium for most of them anyway?

@peduarte happy with that too?

Reply:
> Ok @vladmoroz , so just so I understand what I need to change here then, are we saying remove the fontWeight variant alltogether and update all the cases where it was used to remove it because once your PR goes in, it'll be medium for most of them anyway?

Yeah, you are correct. Here's what's need to be done after merging #226
- Remove `fontWeight={500}` from Heading instances
- Any headings that didn't use `fontWeight` should get `sx={{ fontWeight: 400 }}`
  - Newer designs are expected to use medium headings, but rare exceptions are okay

Reply:
So just to summarize an off conversation on Slack between Pedro, Vlad and I:

- `Text` and `Heading` will now have a `weight: 'normal' | 'medium'` variant
- `Text` default `weight` will be `'normal'`
- `Heading` default `weight` will be `'medium'`